### ISSUE TITLE
Columns sets 

### ISSUE THREAD

**Issue by TomaszGasior** [TomaszGasior/RadioLista-v3#121]
Introduce columns sets. User is able to set custom columns order and visibility. But there are a lot of colums now. With predefined sets user will have better start and easier organization.

**Comment by HyperDX** [comment]
Well I think "must have" is
Channel
Frequency
MUX Name, Package Name (?)
MUX content (List of stations)
And some other columns which are implemented: distance, country, location etc.)

**Comment by TomaszGasior** [comment]
My idea is to collect here some ideas about how these "sets" would actually be organized. I would like to keep ability to define columns custom order and visibility. But there are a lot of columns and probably there will be more in the future. What I call "set" is named predefined collection of columns like "DAB bandscan" or "FM bandscan" or "private list" or "regular reception". After choosing one set, user will still be able to customize order and visibility of each column. The set is just some kind of starting point to make configuration easier.

@HyperDX Basing on your experience, could you please provide some ideas about these sets, some concrete examples? :)

**Comment by HyperDX** [comment]
Ok, so here are my basic sets

**DAB bandscan (only one scan):**
Frequency
Channel
Polarization
MUX Name
Country
Transmitter location
ERP Power (kW)
Distance

**FM bandscan (for only one scan):**
Frequency
Polarization
Station name
Country
Transmitter location
RDS
ERP Power (kW)
Distance

**Private list (Tropo-log)**
For FM
Frequency
Polarization
Station name
Country
Transmitter location
RDS
PI
ERP Power (kW)
Distance
1st log date
Reception type

For DAB
Frequency
Channel
Polarization
MUX Name
Country
Transmitter location
ERP Power (kW)
Distance
1st log date
Reception type

**Regular reception**
For DAB
Frequency
Channel
MUX Name
Country
Transmitter location

For FM
Frequency
Station name
Country
Transmitter location
RDS

**Comment by TomaszGasior** [comment]
This may be implemented as part of new design.

**Comment by TomaszGasior** [comment]
I postponed this feature. It will be implemented after #213
